presto-orc/src/test/java/io/prestosql/orc/BenchmarkColumnReaders.java

```diff
@@ -347,6 +239,15 @@ private Object readFirstColumn(OrcRecordReader recordReader)
     public abstract static class TypeBenchmarkData
             extends BenchmarkData
     {
+        @Param({"0", ".5"})
```

Comment:
Maybe we should simulate all null case with `1.01` chance (instead of having `AllNullBenchmarkData`)?

Reply:
1.0 will work. And yes, I think it is a good idea.